src/main/java/frc/robot/scoreStates/AmpState.java

```diff
@@ -73,23 +61,11 @@ public Command driveToClosestOptimalPoint() {
 
     @Override
     public Command score() {
-        return Commands.sequence(
-                driveToClosestOptimalPoint(),
-                Commands.runOnce(() -> SwerveDrive.getInstance().lock()),
-                Commands.parallel(
-                        elevator.setHeight(CommandGroupsConstants.MAX_HEIGHT),
-                        Commands.either(
-                                gripper.setRollerAndWrist(
-                                        GripperConstants.AMP_POWER_NORMAL,
-                                        CommandGroupsConstants.WRIST_ANGLE_AMP_FORWARD),
-                                gripper.setRollerAndWrist(
-                                        GripperConstants.AMP_POWER_REVERSE,
-                                        CommandGroupsConstants.WRIST_ANGLE_AMP_BACKWARDS),
-                                () -> isAmpingForward)));
+        return null;
```

Comment:
It is better to return `Commands.none()`, so in the rare case `score()` will be scheduled, the code won't crash because you'll try to run a `null` command .

Reply:
b6110bb35c20abf310b1f31b2097ec84e007a7ad